### PR TITLE
Format all rule files

### PR DESCRIPTION
**Pull request by davkean**
Formats all rule files against settings in .editorconfig.

### CODE REVIEW

**Comment by davidwengier** [comment]
Could be GitHub display issues, but it looks like `COMReference.xaml` has been added, but no deletion of `ComReference.xaml`. Additionally do we need to rename all of the .xlf files to pick up the case change?

**Comment by davkean** [comment]
Ugh, no this looks like some sort of bug - I thought I had removed this but VS is picking it up as COMReference, and if I save that - causes above. Need to investigate.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/UpToDateCheckBuilt.xaml [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-    </Rule.DataSource>
-    <BoolProperty Name="Visible" Visible="False" ReadOnly="True" />
-    <StringProperty Name="Original" />    <!-- If set, specifies that this built output comes from this source file. -->
+  <Rule.DataSource>
+    <DataSource Persistence="ProjectFile" HasConfigurationCondition="False" SourceOfDefaultValue="AfterContext"
+                SourceType="TargetResults" MSBuildTarget="CollectUpToDateCheckBuiltDesignTime"/>
+  </Rule.DataSource>
+  <BoolProperty Name="Visible" Visible="False" ReadOnly="True" />
+  <StringProperty Name="Original" />
+  <!-- If set, specifies that this built output comes from this source file. -->
```
Comment (sharwell) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/UpToDateCheckBuilt.xaml]: 💡 This comment should be placed before the property

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/COMReference.xaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Rule Name="ComReference" DisplayName="COM Reference" PageTemplate="generic" Description="COM Reference Properties" xmlns="http://schemas.microsoft.com/build/2009/properties">
```
Comment (sharwell) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/COMReference.xaml]: ❔ Where did this come from?

**Pull request merged**